IGraphics/Platforms/IGraphicsIOS_view.mm

```diff
@@ -832,6 +836,22 @@ - (void) onRotateGesture: (UIRotationGestureRecognizer*) recognizer
   mGraphics->OnGestureRecognized(info);
 }
 
+- (void) onHoverGesture: (UIHoverGestureRecognizer*) recognizer
+{
+  CGPoint pos = [recognizer locationInView:self];
+
+  IMouseInfo info;
+  
+  auto ds = mGraphics->GetDrawScale();
+  
+  info.ms.L = true;
```

Comment:
Is there a reason to set this to true? Normally both L and R would be up when getting mouseovers?

At some point we should also add keyboard input (there's something on a branch somewhere) and set other modifiers correctly.